src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingIdnaConformanceTests.cs

```diff
@@ -13,16 +13,22 @@ namespace System.Globalization.Tests
     /// </summary>
     public class IdnMappingIdnaConformanceTests
     {
+        /// <summary>
+        /// Tests positive cases for GetAscii.  Windows fails by design on some entries that should pass.  The recommendation is to take the source input
+        /// for this if needed.
+        /// 
+        /// There are some others that failed which have been commented out and marked in the dataset as "GETASCII DOES FAILS ON WINDOWS 8.1"
+        /// Same applies to Windows 10 >=RS2 in the IdnaTest_9.txt file
```

Comment:
>RS2 [](start = 41, length = 3)

nit: usually this is internal name. I would put a version (or public release name) instead of using RS2 here or in other places.